src/libsyntax_pos/lib.rs

```diff
@@ -1024,18 +1023,18 @@ impl Decodable for SourceFile {
                 Ok(lines)
             })?;
             let multibyte_chars: Vec<MultiByteChar> =
-                d.read_struct_field("multibyte_chars", 7, |d| Decodable::decode(d))?;
```

Comment:
And same with this change (git blame)

Reply:
Same commit as above. This is just a reordering of the indices to make it more readable/sane.